tests/test_celery.py

```diff
@@ -256,3 +257,13 @@ def test_method_signatures(celery_app, async_task, method, _value):
     if method == "run":
         return
     assert inspect.signature(getattr(async_task.__class__, method)) == inspect.signature(getattr(Task, method))
+
+
+def test__get_backend_returns_DisabledBackground_object_when_result_backend_is_set_to_None(notify_celery):
+    notify_celery.conf.update({"result_backend": None})
```

Comment:
Should test that this works with both the value set to `None` and the config not set at all